### PR TITLE
Feature/wish list form

### PR DESCRIPTION
**Pull request by Satybaldyeva**

### CODE REVIEW

File: src/components/UploadImage.jsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const StyledContainer = styled('div')({
    border: '1px solid #DCDCE4',
    textAlign: 'center',
    gap: '20px',
-   width: '19%',
+   // width: '19%',
```
Comment (NuraiymMamatova) [inline src/components/UploadImage.jsx]: Please delete this comment.

File: src/utils/helpers/validate.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as yup from 'yup'
+
+export const wishListSchema = yup.object().shape({
+   holidayName: yup.string().required('Поле не должен быть пустым!'),
```
Comment (NuraiymMamatova) [inline src/utils/helpers/validate.js]: Sorry, but could you change word in message 'должен' to 'должно'.

File: src/components/UI/SelectComponent.jsx [hunk 3]
```diff
@@ -2,38 +2,67 @@ import {
...
+      control,
+      name,
+      defaultValue: '',
+   })
    return (
-      <StyledFormControl fullWidth>
-         <InputLabel>{label}</InputLabel>
-         <StyledSelect onChange={onChange} value={value} label={label}>
+      <FormControl fullWidth>
+         <FormLabel>{label}</FormLabel>
```
Comment (NuraiymMamatova) [inline src/components/UI/SelectComponent.jsx]: label also should to be red in error

File: src/pages/LandingPage/WishListForm.jsx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+                  data={options}
+                  label="Праздник"
+                  isButton="true"
+                  placeholder="Выберите праздник "
+                  name="holiday"
+                  control={control}
+                  error={Boolean(errors.holiday)}
+                  helperText={errors.holiday?.message}
+               />
+               <StyledDatePicker
```
Comment (NuraiymMamatova) [inline src/pages/LandingPage/WishListForm.jsx]: [Screencast from 2023-11-02 13-01-56.webm](https://github.com/peaksoft-school/gift-list-js10/assets/110243235/5df5ff56-d9fa-460c-8f53-7a9252092712)

Comment (NuraiymMamatova) [reply src/pages/LandingPage/WishListForm.jsx]: and please write name in English.

**Review by Kacym — APPROVED** [review]

File: src/components/DatePicker.jsx [hunk 5]
```diff
@@ -45,13 +52,24 @@ export const DatePicker = ({
...
    ...rest
 }) => {
-   const { field } = useController({ control, name, defaultValue: null })
+   const contr = useController({
+      control,
+      name,
+      defaultValue: null,
+   })
+   const { field } = contr
+   console.log(contr)
```
Comment (NuraiymMamatova) [inline src/components/DatePicker.jsx]: please delete log

File: src/components/UI/SelectComponent.jsx [hunk 6]
```diff
@@ -2,52 +2,116 @@ import {
...
+                  selected
+               )
+            }}
+         >
+            {data.map((title) => {
                return (
-                  <SelectContainer key={i.id} value={i.text}>
-                     {i.text}
-                  </SelectContainer>
+                  <SelectContainer2 key={title} value={title}>
```
Comment (NuraiymMamatova) [inline src/components/UI/SelectComponent.jsx]: Please rename  this tag.

**Review by NuraiymMamatova — APPROVED** [review]

**Pull request merged**